src/ResourceGraph/ResourceGraph/Models/PSResourceGraphResponse.cs

```diff
@@ -14,17 +14,49 @@
 
 namespace Microsoft.Azure.Commands.ResourceGraph.Models
 {
+    using System.Collections;
     using System.Collections.Generic;
-    using System.Management.Automation;
     using Microsoft.WindowsAzure.Commands.Common.Attributes;
 
-    public class PSResourceGraphResponse
+    public class PSResourceGraphResponse<PSObject> : IList<PSObject>
```

Comment:
>PSResourceGraphResponse<PSObject> : IList<PSObject> [](http://example.com/codeflow?start=17&length=51)

@VeryEarly thank you very much for all the help here addressing our cmdlets compatibility issues and formatter concers. You Rock !!!:thumbsup: :rocket: